tests/kernel/mem_protect/mem_protect/src/mem_protect.h

```diff
@@ -30,6 +30,15 @@ static inline void set_fault_valid(bool valid)
 #define MSG_Q_SIZE (10)
 #define MSG_Q_MAX_NUM_MSGS (10)
 #define MSG_Q_ALIGN (2)
+#define PRIORITY 5
+#define STACK_SIZE (512 + CONFIG_TEST_EXTRA_STACKSIZE)
+#define BLK_SIZE_MIN 16
+#define BLK_SIZE_MAX 64
+#define BLK_NUM_MIN 8
```

Comment:
why are these values introduced? what uses them?

Reply:
In tests/kernel/mem_protect/mem_protect/src/inherit.c 
`K_MEM_POOL_DEFINE(res_pool, BLK_SIZE_MIN, BLK_SIZE_MAX, BLK_NUM_MAX, BLK_ALIGN);`
parent_thr and child_thr during k_thread_create use PRIORITY and STACK_SIZE